prereise/gather/winddata/hrrr/tests/test_grib.py

```diff
@@ -0,0 +1,61 @@
+from prereise.gather.winddata.hrrr.grib import GribRecordInfo
+
+GRIB_RECORD_INFO_ARRAY = [
+    "52:38983378:d=2016010121:UGRD:10 m above ground:anl:",
+    "53:40192462:d=2016010121:VGRD:10 m above ground:anl:",
+]
+
+
+def test_grib_info_from_string():
+    g = GribRecordInfo.from_string(GRIB_RECORD_INFO_ARRAY[0])
+    assert g.message_number == "52"
+    assert g.beginning_byte == "38983378"
+    assert g.ending_byte is None
+    assert g.initialization_date == "d=2016010121"
+    assert g.variable == "UGRD"
+    assert g.level == "10 m above ground"
+    assert g.forecast == "anl"
```

Comment:
BTW, since it's a dataclass now, there is the option to do something like this. Feel free to use either approach, just mentioning in case it's useful. 
```
expected = GribRecordInfo(
    message_number="52",
    beginning_byte="38983378",
    ending_byte=None,
    initialization_date="d=2016010121",
    variable="UGRD",
    level="10 m above ground",
    forecast="anl",
)
assert g == expected
```